include/wx/datetime.h

```diff
@@ -392,6 +392,9 @@ class WXDLLIMPEXP_BASE wxDateTime
         // returns true if the given year is a leap year in the given calendar
     static bool IsLeapYear(int year = Inv_Year, Calendar cal = Gregorian);
 
+        // returns the first day of week based on locale and/or OS settings
+    static WeekDay GetFirstWeekDay(bool *success = NULL);
```

Comment:
It doesn't _quite_ fit, but we already have `WeekFlags` enum, wouldn't it make sense to use it, rather than `WeekDay`, here? This would also make `success` parameter unnecessary...